tools/deployment/make_osx_package.sh

```diff
@@ -46,29 +44,39 @@ set -e
 
 POSTINSTALL_ADDITIONAL_TEXT="
 if launchctl list | grep -qcm1 osquery; then
-  launchctl unload $LAUNCHD_INSTALL_PATH
+  launchctl unload $LD_INSTALL
+  launchctl load $LD_INSTALL
 fi
-
-launchctl load $LAUNCHD_INSTALL_PATH
 "
 
 function usage() {
-  fatal "Usage: $0 [-c path/to/your/osquery.conf] [-n] [-l path/to/osqueryd.plist]
+  fatal "Usage: $0 [-c path/to/your/osquery.conf] [-l path/to/osqueryd.plist]
     -c PATH embed an osqueryd config.
-    -n Do not embed the default launchd entry.
-    -l PATH override the default launchd entry."
+    -l PATH override the default launchd plist.
+    -o PATH override the output path.
+
+  This will generate an OSX package with:
+  (1) An example config /var/osquery/osquery.example.config
+  (2) An optional config /var/osquery/osquery.config if [-c] is used
+  (3) A LaunchDaemon plist /var/osquery/com.facebook.osqueryd.plist
+  (4) The osquery toolset /usr/local/bin/osquery*
+
+  To enable osqueryd to run at boot using Launchd use osqueryctl.
```

Comment:
should we add osqueryctl before we add this? or should we just make sure it's there before we cut the next release?